scripts/testnet/include/config.sh

```diff
@@ -94,10 +94,19 @@ prepareElasticsearch() {
     ES_CONTAINER_ID=$(docker start $ES_CONTAINER)
   else
     ES_CONTAINER_ID=$(docker run -d --network host --volume=$ELASTICSEARCH_VOLUME:/usr/share/elasticsearch/data -e "discovery.type=single-node" --name=$ES_CONTAINER docker.elastic.co/elasticsearch/elasticsearch:7.10.2)
+    sleep 20
```

Comment:
Do we really need 20 seconds sleep? This execution has to wait for the script to run anyway

Reply:
I can remove this sleep, it will just show some "retrying..." messages until docker container is ready